### ISSUE TITLE
[4.0] select category buttons

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#28363]
![image](https://user-images.githubusercontent.com/1296369/76735447-d962bb80-675c-11ea-9f07-b4c60f2c236d.png)

The second button should have rounded corners top-right and bottom-right. But the css only applies that rule to the last-child.

In this case its not the last-child because there are two additional children that are hidden
![image](https://user-images.githubusercontent.com/1296369/76736198-475bb280-675e-11ea-955f-63a93c499282.png)

**Comment by infograf768** [comment]
same for the Associations Tab language fields.
Can solve in LTR
```
.input-group-append .btn + .btn {
  [dir=ltr] & {
    @include border-right-radius($border-radius !important);
  }
}
```
not in RTL.

**Comment by brianteeman** [comment]
That scss doesn't work when the active buttons change eg

![image](https://user-images.githubusercontent.com/1296369/77146588-476ff100-6a83-11ea-94dc-07c4654ec12c.png)

**Comment by brianteeman** [comment]
~worked it out - pr  incoming~

**Comment by brianteeman** [comment]
closed please test #28404
